test/test_jit_cuda_fuser.py

```diff
@@ -1750,6 +1750,18 @@ def t(x: torch.Tensor, p: float, train: bool):
         # numbers between eager mode and the jit is different
         self._run_training_helper(t_jit, t, grads, x, 0.0, True)
 
+        def t2(x: torch.Tensor, p: float, train: bool):
+            o = torch.nn.functional.softmax(x, dim=-1)
+            o = torch.nn.functional.dropout(o, p, training=train)
+            return o
+
+        t2_jit = torch.jit.script(t2)
```

Comment:
just for my own education: what is the reason for not using @torch.jit.script decorator directly on t2?

Reply:
there's no good reason for that. I just never default to decorator (not a sophisticated pythoner)